src/pages/WhitePaper/WhitePaperCard/index.tsx

```diff
@@ -30,56 +30,54 @@ interface WhitePaperCardProps {
 }
 function WhitePaperCard({ whitePaper }: WhitePaperCardProps) {
   const isWhitePaperGreen = whitePaper === "technology";
-  const [isLangSelectorOpen, setLanguageSelector] = useState(true);
+  const [isLanguageDropdownOpen, setLanguageDropdown] = useState(true);
   const [language, setLanguage] = useState("English");
 
   return (
     <div
+      onClick={() => setLanguageDropdown((prev) => !prev)}
```

Comment:
sorry to be picky, can this be 'setDropdownOpen' ?

Reply:
Sure! that sounds way better!